src/main/java/uk/gov/hmcts/reform/orgrolemapping/domain/service/RequestMappingService.java

```diff
@@ -32,20 +38,33 @@
 import java.util.Objects;
 import java.util.Set;
 import java.util.UUID;
+import java.util.concurrent.ConcurrentHashMap;
 import java.util.concurrent.atomic.AtomicInteger;
 
 @Service
 @Slf4j
 @AllArgsConstructor
+@NoArgsConstructor
```

Comment:
Why we need @NoArgsConstructor?

Reply:
This is to satisfy the integration tests and when it load the entire context it try to create object and throws "NoSuchBeanDefinitionException" We can discuss, if required